js_modules/dagster-ui/packages/ui-core/src/automation/VirtualizedAutomationSensorRow.tsx

```diff
@@ -132,15 +140,32 @@ export const VirtualizedAutomationSensorRow = forwardRef(
           </RowCell>
           <RowCell>
             <Box flex={{direction: 'row', gap: 8, alignItems: 'flex-start'}}>
-              {/* Keyed so that a new switch is always rendered, otherwise it's reused and animates on/off */}
+              {/* Left aligned content */}
+              <Box flex={{grow: 1, gap: 8}}>
+                {/* Keyed so that a new switch is always rendered, otherwise it's reused and animates on/off */}
+                {sensorData ? (
+                  <SensorSwitch key={name} repoAddress={repoAddress} sensor={sensorData} />
+                ) : (
+                  <div style={{width: 30}} />
+                )}
+                <Link to={workspacePathFromAddress(repoAddress, `/sensors/${name}`)}>
+                  <MiddleTruncate text={name} />
+                </Link>
+              </Box>
+              {/* Right aligned content */}
               {sensorData ? (
-                <SensorSwitch key={name} repoAddress={repoAddress} sensor={sensorData} />
+                <EvaluateTickButtonSensorWrapper>
+                  <EvaluateTickButtonSensor
+                    cursor={cursor || ''}
+                    name={sensorData?.name || ''}
+                    repoAddress={repoAddress}
+                    jobName={sensorData?.targets?.[0]?.pipelineName || ''}
+                    sensorType={sensorData.sensorType}
+                  />
+                </EvaluateTickButtonSensorWrapper>
               ) : (
                 <div style={{width: 30}} />
               )}
```

Comment:
If you're trying to align things left/right you should use `justifyContent: 'space-between'`